yoti/YotiWidget.php

```diff
@@ -26,8 +26,14 @@ public function widget($args, $instance)
     {
         wp_enqueue_style('yoti-asset-css', plugin_dir_url(__FILE__) . 'assets/styles.css');
         $config = YotiHelper::getConfig();
+        $widgetTitleHtml = '';
+        $widgetDefaultTitle = 'New title';
+        // Make sure the title is no empty and is not the default text
+        if(!empty($instance['title']) && $instance['title'] !== $widgetDefaultTitle){
+            $widgetTitleHtml = '<h3 class="yoti-widget-title">' . $instance['title']. '</h3>';
+        }
         if (!empty($config['yoti_sdk_id']) && !empty($config['yoti_pem']['contents'])) {
-            echo '<div class="yoti-connect-button">' . YotiButton::render(NULL, TRUE) . '</div>';
+            echo '<div class="yoti-connect-button">' . $widgetTitleHtml . YotiButton::render(NULL, TRUE) . '</div>';
```

Comment:
Why not default style: `<section id="yoti-widget" class="widget widget_yoti">`?

Reply:
Same reason as above, I think the gap between the title and the line is way too big and having  our own class I can better customise it. Again if you thing WordPress default display is better that's fine by me too.